tests/stress.rs

```diff
@@ -659,11 +659,9 @@ fn test_dc_mimeparser_with_context() {
         let context = create_test_context();
 
         let mut mimeparser = dc_mimeparser_new(&context.ctx);
-        let raw: *const libc::c_char =
-        b"Content-Type: multipart/mixed; boundary=\"==break==\";\nSubject: outer-subject\nX-Special-A: special-a\nFoo: Bar\nChat-Version: 0.0\n\n--==break==\nContent-Type: text/plain; protected-headers=\"v1\";\nSubject: inner-subject\nX-Special-B: special-b\nFoo: Xy\nChat-Version: 1.0\n\ntest1\n\n--==break==--\n\n\x00"
-            as *const u8 as *const libc::c_char;
+        let raw = b"Content-Type: multipart/mixed; boundary=\"==break==\";\nSubject: outer-subject\nX-Special-A: special-a\nFoo: Bar\nChat-Version: 0.0\n\n--==break==\nContent-Type: text/plain; protected-headers=\"v1\";\nSubject: inner-subject\nX-Special-B: special-b\nFoo: Xy\nChat-Version: 1.0\n\ntest1\n\n--==break==--\n\n\x00";
```

Comment:
Would you mind moving this test to a test module in the dc_mimeparser.rs file?  I think we should aim to remove this monolithic stress test and break it out into unittests.

Reply:
Done.